Sources/BraintreeVenmo/Models/VenmoAccountsRequest.swift

```diff
@@ -0,0 +1,14 @@
+/// The POST body for `v1/payment_methods/venmo_accounts`
+struct VenmoAccountsRequest: Encodable {
```

Comment:
Yeah I think that makes sense, I think we just rolled with request because that's what the first PR had, but agree it's a bit confusing. I think `body` is more accurate, we could even take it a step further and do `VenmoAccountsPOSTBody` if we think that'd be more detailed/specific.

Reply:
I'm down for that! Agreed on doing it in a separate PR 👍

Reply:
Ticket for the renames: DTMOBILES-1202